patterns/store-info-alt-image-and-text.php

```diff
@@ -13,7 +20,7 @@
 		<div class="wp-block-column is-vertically-aligned-center" style="flex-basis:50%">
 			<!-- wp:image {"sizeSlug":"full","linkDestination":"none"} -->
 			<figure class="wp-block-image size-full">
-				<img src="<?php echo esc_url( plugins_url( 'images/pattern-placeholders/crafting-pots.png', dirname( __FILE__ ) ) ); ?>" alt="<?php esc_attr_e( 'Placeholder image used to represent a person making a clay pot.', 'woo-gutenberg-products-block' ); ?>" />
+				<img src="<?php echo esc_url( $image1 ); ?>" alt="<?php esc_attr_e( 'Placeholder image used to represent a person making a clay pot.', 'woo-gutenberg-products-block' ); ?>" />
```

Comment:
Am I understanding correctly that all these images will be generated based on store description? I'm just wondering if the image is unrelated due to store description won't the alt text be very confusing?

Reply:
It's true, now that we don't know the actual images that will be displayed, I think we should make the `alt` description more generic. I'll work on it. Thanks 🙌